includes/definitions/discovery/ecreso-fm-tx.yaml

```diff
@@ -0,0 +1,119 @@
+mib: ECRESO-FM-TRANS-MIB
+modules:
+  os:
+    hardware: ECRESO-FM-TRANS-MIB::syGeHardRel.0
+    serial: ECRESO-FM-TRANS-MIB::syGeSerial.0
+    version: ECRESO-FM-TRANS-MIB::syGeSoftRel.0
+  sensors:
+    power:
+      data:
+        - oid: txGnMsForwardPower
+          num_oid: ".1.3.6.1.4.1.6404.3.1.2.7.1.3.1.0"
+          descr: "Forward Power (W)"
+          index: 1
```

Comment:
Index should typically be based on the old, I.e: `index: 'txGnMsForwardPower.{{ $index }}'`

Please set this for all sensors.